tool/tsh/tsh.go

```diff
@@ -1953,6 +1996,22 @@ func makeClient(cf *CLIConf, useProfileLogin bool) (*client.TeleportClient, erro
 		return nil, trace.Wrap(err)
 	}
 
+	if c.ExtraProxyHeaders == nil {
+		c.ExtraProxyHeaders = map[string]string{}
+	}
+	for _, proxyHeaders := range cf.ExtraProxyHeaders {
+		proxyGlob := utils.GlobToRegexp(proxyHeaders.Proxy)
+		proxyRegexp, err := regexp.Compile(proxyGlob)
+		if err != nil {
+			return nil, trace.WrapWithMessage(err, "Invalid proxy glob %s", proxyGlob)
```

Comment:
Nit: Do not capitalize returned errors.
```suggestion
			return nil, trace.Wrap(err, "invalid proxy glob %q in tsh configuration file", proxyGlob)
```